example/arcfullsim.py

```diff
@@ -0,0 +1,134 @@
+"""
+DD4hep simulation with some argument parsing
```

Comment:
This comment is not what this file is

Reply:
Thanks for the review, I have updated the header of the file